File: Find Numbers with Even Number of Digits 1295/solution.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int result = 0;
        int count = 1;
        for ( vector<int>::iterator it = nums.begin(); it != nums.end(); ++it )
        {
            while (*it / 10)
            {
                count++;
            }
            
            if ( 0 == (count % 2) )
                result++;
        }
        
        return result;
    }
};

File: Find Numbers with Even Number of Digits 1295/solution.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        length = len(nums)
        count = 1
        result = 0
        for i in range(length):
            num = nums[i]
            num = num // 10
            while ( num > 0 ):
                num = num // 10
                count = count + 1
               
            temp = count % 2
            if ( 0 == temp ):
                result = result + 1
                
            count = 1
                
              
          
        return result